taskt/Core/Automation/Commands/Excel Commands/ExcelDeleteRangeCommand.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.Office.Interop.Excel;
+using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
+using Application = Microsoft.Office.Interop.Excel.Application;
+
+namespace taskt.Core.Automation.Commands
+{
+    [Serializable]
+    [Group("Excel Commands")]
+    [Description("This command deletes a specific cell in an Excel Worksheet.")]
```

Comment:
Please update description as this command deletes a range not a specific cell.

Reply:
done